submissions/vasylpaliy/html_css_popup/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" href="style.css">
+   <title>popup</title>
+</head>
+
+<body>
+   <header>
+      <nav>
+         <ul class="container-nav">
+            <li>
+               <a href="#" class="container-nav-link ">Mail</a>
+            </li>
+            <li>
+               <a href="#" class="container-nav-link ">Images</a>
+            </li>
+            <li>
+               <input type="checkbox" id="my-popup" class="input-nav-popup">
+               <label for="my-popup" class="container-label">
+                  <img src="icons/popup-button.png" alt="Open popup menu" class="container-navbar-img">
+               </label>
+               <div class="popup-container">
+                  <ul class="popup">
+                     <li>
+                        <a href="#" class="popup-text ">
+                           <img src="icons/calendar.png" class="popup-img">
```

Comment:
The `alt` attribute is the required image attribute, if you want to leave it empty use this example:
```HTML
<img alt="" />
```